feature/security/gnsi/certz/tests/README.md

```diff
@@ -0,0 +1,60 @@
+# gNSI TLS Authentication Testing
+
+## Summary
+Test gRPC TLS authentication under various conditions created using
+combinations of different security credentials and artifacts managible using
+the gNSI protocol, including device certificates, Trust-Bundle (TB),
+Certificate Revocation List (CRL) and Authentication Policy.
+
+## Authentication Policy Enforcement
+
+Test device enforcement of
+[`gnsi.certz.v1.AuthenticationPolicy`](https://github.com/openconfig/gnsi/blob/main/certz/certz.proto)
+against clients connecting with different certificates.  Repeat all test cases
+for every TLS secured service.
+
+Testing of `AuthenticationPolicy` rotation over the gNSI protocol is not within
+the scope of this section.
+
+### Input Args {#input-args}
+
+* `testdata/leaf_cert.pem`: valid client certificate
+* `testdata/leaf_key.pem`: private key for `leaf_cert.pem`
+* `testdata/bad_leaf_cert_1.pem`: client certificate issued by unauthorized issuer
+* `testdata/bad_leaf_key_1.pem`: private key for `bad_leaf_cert_1.pem`
+* `testdata/bad_leaf_cert_2.pem`: client certificate containing inconsistent trust domains
+* `testdata/bad_leaf_key_2.pem`: private key for `bad_leaf_cert_2.pem`
+* `testdata/bad_leaf_cert_3.pem`: client certificate containing inconsistent realm
+* `testdata/bad_leaf_key_3.pem`: private key for `bad_leaf_cert_3.pem`
+
+### DUT service setup
+
+Setup all gRPC services of DUT with the followings:
+* `testdata/leaf_cert.pem`: server certificate
+* `testdata/leaf_key.pem`: private key for `leaf_cert.pem`
+* `testdata/root_cert.pem`: trust bundle
+* `testdata/policy_1.pb`: authentication policy that client certificates are tested against
+* `loas3_disable_realm_consistency_check`: Server flag set according to [Tests](#tests) below.
+
+The following files are included for debugging purposes only:
+* `testdata/policy_1.textproto`: textual format of `policy_1.pb`
```

Comment:
it's not clear to me why we'd provide a binary proto here for the test, why not just stick to textproto which is inspectable? (instead of requiring future changes to change 2 files)

Reply:
Use of proto in text format should be avoided because the its parser is dangerously forgiving.

https://protobuf.dev/programming-guides/dos-donts/#dont-use-text-format-messages-for-interchange

Reply:
ok thanks!